### ISSUE TITLE
无法传送 mp4、zip 文件

### ISSUE THREAD

**Issue by HaujetZhao** [YanxinTang/clipboard-online#30]
经短暂使用测试，能够传的后缀名似乎是有限的。jpg、kbdx 都可以上传、下载，但 mp4、zip 格式都会出问题，无法正常通信。

**Comment by HaujetZhao** [comment]
不对。是我的文件太大了。8MB 的视频文件，才传不了的，压缩后也传不了。应该是返回值的大小限制了。这一点可以在 readme 中说明。

**Comment by HaujetZhao** [comment]
对这个8MB 的文件，我在捷径 App 里运行复制，可以成功。但是从桌面、Siri 运行时，就会失败，无法成功复制。

**Comment by YanxinTang** [comment]
多谢您的建议，目前捷径复制稍大的文件确实会失败，后续会加到 readme 中进行说明。

**Comment by Christine0801** [comment]
文件太大了，Base64解码，会超出长度吧？应该是这个原因
